CoreErrorReportView/src/au/gov/asd/tac/constellation/views/errorreport/ErrorReportTopComponent.java

```diff
@@ -107,14 +105,35 @@ public class ErrorReportTopComponent extends JavaFxTopComponent<BorderPane> {
     private List<ErrorReportEntry> hiddenErrors = new ArrayList<>();
     static VBox sessionErrorsBox = new VBox();
     private MenuButton popupControl;
-    private final CheckBox severePopCheckBox = new CheckBox("Allow SEVERE Popups");
-    private final CheckBox severeRepCheckBox = new CheckBox("Display SEVERE Reports");
-    private final CheckBox warningPopCheckBox = new CheckBox("Allow WARNING Popups");
-    private final CheckBox warningRepCheckBox = new CheckBox("Display WARNING Reports");
-    private final CheckBox infoPopCheckBox = new CheckBox("Allow INFO Popups");
-    private final CheckBox infoRepCheckBox = new CheckBox("Display INFO Reports");
-    private final CheckBox finePopCheckBox = new CheckBox("Allow FINE Popups");
-    private final CheckBox fineRepCheckBox = new CheckBox("Display FINE Reports");
+    public enum SeverityCode {
+        SEVERE("SEVERE"),
+        WARNING("WARNING"),
+        INFO("INFO"),
+        FINE("FINE");
+        private final String code;
+        SeverityCode(final String severityCode){
+            code = severityCode;
+        }
+        public String getCode(){
+            return code;
+        }
+        public static SeverityCode getSeverityCodeEntry(final String severityCode){
+            for (SeverityCode sevCode : SeverityCode.values()) {
+                if (severityCode.equals(sevCode.getCode())) {
+                    return sevCode;
+                }
+            }
+            return null;
+        }
+    }
+    private final CheckBox severePopCheckBox = new CheckBox("Allow " + SeverityCode.SEVERE.getCode() + " Popups");
+    private final CheckBox severeRepCheckBox = new CheckBox("Display " + SeverityCode.SEVERE.getCode() + " Reports");
+    private final CheckBox warningPopCheckBox = new CheckBox("Allow " + SeverityCode.WARNING.getCode() + " Popups");
+    private final CheckBox warningRepCheckBox = new CheckBox("Display " + SeverityCode.WARNING.getCode() + " Reports");
+    private final CheckBox infoPopCheckBox = new CheckBox("Allow " + SeverityCode.INFO.getCode() + " Popups");
+    private final CheckBox infoRepCheckBox = new CheckBox("Display " + SeverityCode.INFO.getCode() + " Reports");
+    private final CheckBox finePopCheckBox = new CheckBox("Allow " + SeverityCode.FINE.getCode() + " Popups");
+    private final CheckBox fineRepCheckBox = new CheckBox("Display " + SeverityCode.FINE.getCode() + " Reports");
```

Comment:
I'm not sure there's enough value in creating a private method which contains a single line of code. It won't be useable outside of the class and it won't reduce the lines of code. All it will do is add another method to the call stack.